seml/database.py

```diff
@@ -3,11 +3,15 @@
 from pymongo.collection import Collection
 import logging
 from tqdm.auto import tqdm
+import re
+from collections import Counter
```

Comment:
We technically do not need the Counter anymore, but you use it here as a `defaultdict`.